src/current/advisories/a104309.md

```diff
@@ -80,6 +80,32 @@ If data is found, it can be re-emitted in two ways:
 - By restarting the changefeed with `initial_scan` set to `yes` and without a cursor timestamp. This will perform a full export of all current data across the changefeed.
 - By restarting the changefeed and providing a cursor timestamp. If a cursor timestamp is provided, an initial scan will not be performed. Instead, updates above the cursor timestamp will be emitted. The cursor timestamp can be initialized below the timestamp of omitted writes (refer to the reference documentation for [CREATE CHANGEFEED](https://www.cockroachlabs.com/docs/stable/create-changefeed)). However, specifying a cursor timestamp farther in the past than the [`gc.ttlseconds`](https://www.cockroachlabs.com/docs/stable/configure-replication-zones#gc-ttlseconds) replication zone variable, which defaults to `14400` seconds (4 hours), results in an error.
 
+{{site.data.alerts.callout_info}}
+This fix introduces a [bug](https://github.com/cockroachdb/cockroach/issues/119536) that could cause a rangefeed’s resolved timestamp to stop advancing. The corresponding changefeed will appear to be stalled in `RUNNING` state in certain conditions: If a rangefeed is running on a follower on a recently-merged range and encounters an aborted transaction, then the resolved timestamp may stall if the node has a stale range descriptor cache. Events will still be emitted as normal.
+
+After upgrading to a version with the fix for this advisory, **a104309**, your cluster may encounter a stalled rangefeed or changefeed even if it was not impacted by the original rangefeed bug. Monitor your cluster using the following mechanisms:
+
+- Monitor cluster logs for messages like:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ none
+    pushing old intents failed: range barrier failed, range split
+    ~~~
+
+- Monitor the following cluster metrics:
```

Comment:
Link to https://www.cockroachlabs.com/docs/stable/monitor-and-debug-changefeeds may be useful here